File: Assignment8/C/Program2.c
// PS - Write a program which acepts one number from user and return count of odd digits

#include<stdio.h>

int CountOdd(int iNo) {
    int iDigit = 0;
    int iCount = 0;

    if(iNo < 0) {
        iNo = -iNo;
    }

    while(iNo != 0) {
        iDigit = iNo % 10;
        if((iDigit % 2) != 0){
            iCount++;
        }
        iNo = iNo / 10;
    }
    return iCount;
}

int main() {
    int iValue = 0;
    int iRet = 0;

    printf("Enter number \n");
    scanf("%d", &iValue);

    iRet = CountOdd(iValue);

    printf("Count of odd digits are %d \n", iRet);

    return 0;
}

File: Assignment35/Java/Program2.java
// PS - Write a java program which accept number from user and return count of odd digits

import java.util.*;

class DigitDemo
{
    public int CountOdd(int No)
    {
        int Odd = 0;
        int iDigit = 0;

        while(No != 0)
        {
            iDigit = No % 10;
            if((iDigit % 2) != 0)
            {
                Odd++;
            }
            No = No / 10;
        }
        return Odd;
    }
}

class Program2
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int No = sobj.nextInt();

        DigitDemo dobj = new DigitDemo();
        int iRet = dobj.CountOdd(No);

        System.out.println("Count of odd digits are : "+iRet);
    }
}